shared/actions/favorite.js

```diff
@@ -122,6 +122,45 @@ let previousNotifyState = null
 
 const injectMeta = type => f => { f.meta = type }
 
+const jsonToFolders = (json: Object, myKID: any) => {
```

Comment:
🐑 a type annotation for the output would be great so we can skim the type and understand the function faster